src/Rector/Contrib/PHPUnit/SpecificMethod/AssertCompareToSpecificMethodRector.php

```diff
@@ -81,13 +82,14 @@ public function isCandidate(Node $node): bool
             return false;
         }
 
-        $secondArgumentValue = $methodCallNode->args[1]->value->name;
+        /** @var Expr $secondArgumentValue */
+        $secondArgumentValue = $methodCallNode->args[1]->value;
 
-        if (! $secondArgumentValue instanceof Name) {
+        if (! $this->isNamedFunction($secondArgumentValue)) {
             return false;
         }
 
-        $methodName = $secondArgumentValue->getFirst();
+        $methodName = $secondArgumentValue->name->getFirst();
```

Comment:
`$secondArgumentValue->name` is `Identifier`, right?

so 

```php
$secondArgumentValue->name->toString();
``` has to be used.

If that doesn't help, then:

```php
(string) $secondArgumentValue->name;
```

Reply:
Yes, you're right. Only in `isNamedFunction` we need to check for `Name`. Here, is `Identifier`.